apple/macbook-pro/12-1/README.md

```diff
@@ -0,0 +1,12 @@
+# MacBook Pro 12,1
+
+## Wireless won't get reconnected after resume/hibernate
+
+As an example similar code could be used to restart wireless interface:
+
+```nix
+powerManagement.powerUpCommands = ''
```

Comment:
@Infinisil How can we make this string appended after our own module value? We are currently doing `rmmod`. I think there was a way to do priorities.

Reply:
Not sure how `rmmod` relates, but if you mean how to order different `powerUpCommand` assignments, you can use `mkBefore` or `mkAfter` for that, which make sure the value is ordered before or after the default priority.

Reply:
Thanks! This was the helper functions I was looking for.

Reply:
@sevenfourk If you revert to the previous `powerManagement.powerUpCommands/powerDownCommands`
but use `mkBefore`:

```
powerManagement.powerUpCommands = lib.mkBefore ''
  ${pkgs.kmod}/bin/modprobe brcmfmac
'';
```

Than users can just follow the readme to restart their network manager daemon of choice and the ordering will make sure that their services gets added after the modprobe.

Reply:
thanks guys, leaving ordering for README example as is. added lib.mkBefore for module manipulation.